src/lib/core/ripple/ripple-renderer.ts

```diff
@@ -1,27 +1,9 @@
-import {
-  ElementRef,
-  NgZone,
-} from '@angular/core';
-
-/** @docs-private */
-export enum ForegroundRippleState {
-  NEW,
-  EXPANDING,
-  FADING_OUT,
-}
+import {ElementRef, NgZone} from '@angular/core';
+import {ViewportRuler} from '../overlay/position/viewport-ruler';
 
-/**
- * Wrapper for a foreground ripple DOM element and its animation state.
- * @docs-private
- */
-export class ForegroundRipple {
-  state = ForegroundRippleState.NEW;
-  constructor(public rippleElement: Element) {}
-}
 
-const RIPPLE_SPEED_PX_PER_SECOND = 1000;
-const MIN_RIPPLE_FILL_TIME_SECONDS = 0.1;
-const MAX_RIPPLE_FILL_TIME_SECONDS = 0.3;
+export const RIPPLE_SPEED_PX_PER_SECOND = 170;
+export const RIPPLE_FADE_OUT_DURATION = 600;
```

Comment:
Add doc somewhere to mention the fade out duration is fixed and is not changed with speed factor?